pkgs/sdk/server-ai/src/LdAiClient.cs

```diff
@@ -73,10 +73,27 @@ public ILdAiConfigTracker ModelConfig(string key, Context context, LdAiConfig de
         }
 
 
-        var prompt =
-            parsed.Prompt?.Select(m => new LdAiConfig.Message(InterpolateTemplate(m.Content, mergedVariables), m.Role));
+        var prompt = new List<LdAiConfig.Message>();
+
+        if (parsed.Prompt != null)
+        {
+            for (var i = 0; i < parsed.Prompt.Count; i++)
+            {
+                try
+                {
+                    var content = InterpolateTemplate(parsed.Prompt[i].Content, mergedVariables);
+                    prompt.Add(new LdAiConfig.Message(content, parsed.Prompt[i].Role));
+                }
+                catch (Exception ex)
+                {
+                    _logger.Error($"AI model config prompt has malformed message at index {i}: {ex.Message} (returning default config)");
```

Comment:
I think we should extend the message with a warning about the interpolation. "The returned configuration will not have interpolated prompts." Or something that indicates that not only is it the default, but it is not interpolated.

Because typically the default will be interpolated (because it goes into and out of variation).

Reply:
Ah, that's a good point.